nav2_mission_executor/README.md

```diff
@@ -0,0 +1,82 @@
+# Mission Executor
+
+The Mission Executor module is a task server that coordinates other tasks. It can be used to direct the activities multiple robots as well as other, non-robot tasks.
```

Comment:
activities OF multiple robots (missing word)

Reply:
```suggestion
The Mission Executor module is a task server that coordinates other tasks. It can be used to direct the activities of multiple robots as well as other, non-robot tasks.
```

I mostly just wanted to try out this feature of Github